src/store/generator/slices/rules.ts

```diff
@@ -4,6 +4,7 @@ import { ImmerStateCreator } from '@/utils/typescript'
 interface State {
   rules: TestRule[]
   selectedRuleId: string | null
+  rulesEnabled: boolean
```

Comment:
I think it should have a different name, because we already have a concept of disabled rules and it means something else.